web/styles/left_sidebar.css

```diff
@@ -203,7 +203,7 @@ li.show-more-topics {
        in the DM section. */
     margin-right: 12px;
 
-    #direct-messages-section-header {
+    &#direct-messages-section-header {
```

Comment:
This is kinda a weird selector; probably we could move this next block to just be a top-level element block in `left_sidebar.css` to clean this up? 

But we can leave that as a follow-up, I don't want to churn this PR unnecessarily since there's another stacked on it.